app/templates/admin/vehicles/upsert.html

```diff
@@ -0,0 +1,142 @@
+{% extends 'admin/base.html' %}
+{% set page_title='Vehicles' %}
+
+{% block action_buttons %}
+  <a href="/admin/vehicles" class="btn btn-link">Back to list</a>
+{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col-8">
+      <form id="vehicleForm" action="/admin/vehicles/save" method="POST" enctype="multipart/form-data">
+        {% if vehicle %}
+          <input name="vehicle_id" type="hidden" value="{{ vehicle['id'] }}"/>
+        {% endif %}
+
+        <div class="mb-3">
+          <label class="form-label">Title</label>
```

Comment:
can we move the title little bit to the down. 
Also we can reorganize the form little bit, for example we can have:
Make - Model - Year (in one line)

Reply:
Absolutely!

Reply:
This has fixed